### PR TITLE
Replace CY8CKIT_064S2_4343W with CY8CKIT_064B0S2_4343W

### PR DESCRIPTION
**Pull request by Patater**
Mbed OS has dropped support for the prototype board
"CY8CKIT_064S2_4343W". Replace with the production board
"CY8CKIT_064B0S2_4343W".

### CODE REVIEW

File: build_psa_compliance.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
                     "tgt_ff_tfm_musca_a"],
```
Comment (jainvikas8) [inline build_psa_compliance.py:37]: Could we remove this please, in a new commit? `ARM_MUSCA_A1` is not supported on Mbed OS 6.0

File: tfm/targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_064S2_4343W/device/COMPONENT_CM4/device_cfg.h [hunk 2]
```diff
@@ -1,41 +0,0 @@
-/*
- * Copyright (c) 2017-2018 Arm Limited
- * Copyright (c) 2020, Cypress Semiconductor Corporation. All rights reserved.
```
Comment (jainvikas8) [inline tfm/targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_064S2_4343W/device/COMPONENT_CM4/device_cfg.h:3]: This file needs to go back to `CY8CKIT_064B0S2_4343W`

File: build_psa_compliance.py (hunk 1 above)
Comment (Patater) [reply build_psa_compliance.py:37]: Yes, separate PR

File: tfm/targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_064S2_4343W/device/COMPONENT_CM4/device_cfg.h (hunk 2 above)
Comment (Patater) [reply tfm/targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_064S2_4343W/device/COMPONENT_CM4/device_cfg.h:3]: OK

File: build_psa_compliance.py (hunk 1 above)
Comment (Patater) [reply build_psa_compliance.py:37]: Created https://github.com/ARMmbed/mbed-os-tf-m-regression-tests/issues/28

**Comment by Patater** [comment]
Awaiting which target we should actually move to

**Pull request closed without merging**